src/olympia/devhub/tests/test_views_submit.py

```diff
@@ -1734,13 +1737,15 @@ def test_failure_when_version_number_doesnt_match(self):
         assert pq(response.content)('ul.errorlist').text() == (
             "Version doesn't match")
 
+    @override_switch('beta-versions', active=True)
     def test_force_beta_is_ignored(self):
         """Files must have the same beta status as the existing version."""
         self.post(beta=True)
         # Need latest() rather than find_latest_version as Beta isn't returned.
         version = self.addon.versions.latest()
         assert version.all_files[0].status != amo.STATUS_BETA
 
+    @override_switch('beta-versions', active=True)
```

Comment:
Needs a test where `beta-versions` is `False`.

Reply:
I am not sure what a test with no beta versions would look like for this.

Reply:
hmm.  I guess this should never happen as files shouldn't be able to be added to a beta version.